core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala

```diff
@@ -33,14 +33,14 @@ class PreparedStatementArguments extends StatementArguments {
 
   def clearArguments(): Unit = {
     argumentsList = NonEmptyList(mutable.Map.empty)
-    maxIndex = 1
+    maxIndex = 0
   }
 
   override def toStringArray: Array[String] = toArray map { value =>
     Option(value) match {
       case Some(v: String) => v
       case Some(v) => v.toString
-      case None => javaNull
+      case None => "NULL"
```

Comment:
I'm wondering if this constant should be pulled up in a kind of `javaNullString` val.